composer.json

```diff
@@ -5,7 +5,7 @@
     "license": "MIT",
     "require": {
         "php": "^8.1",
-        "behat/behat": "^3.0",
+        "behat/behat": "^3.9",
         "symfony/config": "^4.4 || ^5.4 || ^6.0 || ^7.0"
     },
     "require-dev": {
```

Comment:
PHPCS currently fails on lowest deps, https://github.com/nicwortel/behat-unused-step-definitions-extension/actions/runs/9603334121/job/26486267420#step:5:56
We may need to raise dev dependencies but I'm not sure which: nicwortel/coding-standard ?slevomat/coding-standard, squizlabs/php_codesniffer ?). It's a bit unclear. Any idea @nicwortel ?

Reply:
slevomat/coding-standard 7.0.16 or above fixes it, but maybe this should be fixed by the nicwortel/coding-standard dependency ?

Reply:
@vever001, It's just testing, so we're safe to fix the constraint in this PR

Reply:
I raised the constraint on nicwortel/coding-standard (since it is compatible with PHP 8.1 - 8.3), and removed direct dependencies to slevomat/coding-standard and squizlabs/php_codesniffer (since they are included).
I hope it makes sense.